release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -828,7 +821,7 @@ repository = "local"
   - Introduced pushing to the local repository. Passing `--repository=local` will push the Ballerina archive (.bala) to the local repository. For information about local repository support, see the [Ballerina Packages Changelist](<link>).
 - Run `bal help <command>` to get more information on the command changes.
 
-- CLI Auto Completion
+- CLI Auto-Completion
```

Comment:
```suggestion
- CLI Auto-Completion
```
It is the verb form we have here and we do not need the hyphen.

Reply:
I believe auto-completion requires a hyphen. What do you mean verb form? Here's a usage example - https://npp-user-manual.org/docs/auto-completion/

Reply:
For example, "log in" is the verb form in which we say "Log in to some app". "Log-in" with the hyphen is the noun/adjective form of it in which we say "The log-in will require a password". The same applies here.

Reply:
+1